etabotsite/etabotapp/TMSlib/TMS.py

```diff
@@ -112,17 +117,20 @@ def connect_to_TMS(self, update_tms=True):
                 logging.info(
                     'sending email about connectivity issue to: "{}".'.format(
                         self.username_login))
-
-                msg = MIMEMultipart()
-                msg['From'] = '"ETAbot" <no-reply@etabot.ai>'
-                msg['To'] = self.username_login  # TODO: user.email
-                msg['Subject'] = 'Account {} needs attention.'.format(
-                    self.server_end_point)
-                msg_body = '<html><body><h3>Please log in to https://app.etabot.ai/login \
-    and fix credentials for account {}</h3></body></html>'.format(
-                    self.server_end_point)
-                msg.attach(MIMEText(msg_body, 'html'))
-                user_activation.ActivationProcessor.send_email(msg)
+                if self.username_login is not None or '@' not in self.username:
+                    msg = MIMEMultipart()
+                    msg['From'] = '"ETAbot" <no-reply@etabot.ai>'
+                    msg['To'] = self.username_login  # TODO: user.email
+                    msg['Subject'] = 'Account {} needs attention.'.format(
+                        self.server_end_point)
+                    msg_body = '<html><body><h3>Please log in to https://app.etabot.ai/login \
+        and fix credentials for account {}</h3></body></html>'.format(
+                        self.server_end_point)
+                    msg.attach(MIMEText(msg_body, 'html'))
+                    user_activation.ActivationProcessor.send_email(msg)
+                else:
+                    logging.warning('username is not email - \
+cannot send connectivity issue email')
```

Comment:
please fix the format here.

Reply:
that's by design to comply with PEP8 line length